theme/static/js/bootstrap-formhelpers.js

```diff
@@ -1,5 +1,9 @@
 /**
 * bootstrap-formhelpers.js v2.3.0 by @vincentlamanna
+* modified very slightly by Devin Cowan
```

Comment:
I'm generally against making changes to dependencies directly, and rather override their methods in-memory. But if this is a copy of a script that we intent to keep in this location and no longer upgrade, then I'm ok with it.

Reply:
thanks @Maurier that is a good call but I had some issues extending the bootstrap-form-helper functions. I will push my attempt and comment where I ran into trouble...